notesserver/settings/common.py

```diff
@@ -101,8 +99,8 @@
         'DIRS': [
             # The EdxNotes templates directory is not actually under any app
             # directory, so specify its absolute path.
-            os.path.join(BASE_DIR, 'templates'),
-        ]
+            os.path.join(BASE_DIR, 'templates')
```

Comment:
A trailing comma is desirable.

Reply:
Agree. I guess It was autoformatting, anyway I put it back.